__docs__/wonder-blocks-tabs/navigation-tabs-testing-playtesting.stories.tsx

```diff
@@ -0,0 +1,181 @@
+import * as React from "react";
+import {Meta, StoryObj} from "@storybook/react";
+import {
+    NavigationTabItem,
+    NavigationTabs,
+} from "@khanacademy/wonder-blocks-tabs";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {sizing} from "@khanacademy/wonder-blocks-tokens";
+import Tooltip from "@khanacademy/wonder-blocks-tooltip";
+import {PopoverContent, Popover} from "@khanacademy/wonder-blocks-popover";
+import Link from "@khanacademy/wonder-blocks-link";
+import {HeadingMedium} from "@khanacademy/wonder-blocks-typography";
+
+export default {
+    title: "Packages / Tabs / NavigationTabs / Testing / NavigationTabs - Playtesting",
+} as Meta<typeof NavigationTabItem>;
+
+type Story = StoryObj<typeof NavigationTabItem>;
+
+export const Default: Story = {
+    // We include a base default story so that the docs page can include the story
+    // description properly.
+    render: (args) => {
+        return (
+            <NavigationTabs>
+                <NavigationTabItem {...args}>
+                    <Link href="#link">Navigation tab item</Link>
+                </NavigationTabItem>
+            </NavigationTabs>
+        );
+    },
+    parameters: {
+        chromatic: {disableSnapshot: true},
+    },
+};
+
+/**
+ * When a `Link` component is passed in for the `children` prop,
+ * `NavigationTabItem` will inject props for the `Link`.
+ *
+ * For specific use cases where the `Link` component is wrapped by another
+ * component (like a `Tooltip` or `Popover`), a render function can be used
+ * instead. The render function provides the Link props that should be applied
+ * to the Link component. The Link props contains styles and attributes for
+ * accessibility like `aria-current`.
+ *
+ * This story demonstrates how a render function could be used to wrap a `Link`
+ * in a `NavigationTabItem` with a `Tooltip` and a `Popover`. Please test for
+ * accessibility for your use case, especially around focus management,
+ * keyboard interactions, and screenreader support!
+ *
+ * #### Current screenreader behaviour
```

Comment:
TIL (I think) that JSDoc comments in Storybook fully support Markdown (or MDX) 😮.